app/budget/directives/budgetScenarioToolbar/budgetScenarioToolbar.js

```diff
@@ -7,14 +7,17 @@ let budgetScenarioToolbar = function($window, $location, $routeParams, $rootScop
 		restrict: 'E', // Use this via an element selector <budget-scenario-dropdown></budget-scenario-dropdown>
 		template: require('./budgetScenarioToolbar.html'), // directive html found here:
 		replace: true, // Replace with the template
+		
 		scope: {
-			state: '<'
+			state: '<',
+			printBudget: '&',
```

Comment:
This scope variable appears to not be used, am I missing something?

Reply:
Old code, not using it anymore